modules/understanding-upgrade-channels.adoc

```diff
@@ -7,159 +7,145 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="understanding-upgrade-channels_{context}"]
-= {product-title} Upgrade Channels and Releases
+= Understanding {product-title} upgrade channels
```

Comment:
I'd asked for dropping "Understanding" [here][1].  Is there a reason this particular header needs to call out "Understanding" but not all the other headers in openshift-docs?

[1]: https://github.com/openshift/openshift-docs/pull/19088#discussion_r367762031